linchpin_config.yml

```diff
@@ -2,11 +2,13 @@
 keystore_path: "keystore"
 outputfolder_path: "outputs"
 inventoryfolder_path: "inventory"
-
+# inventory playbooks are not necessary because inventory_type specific tasks are defined in inventory_gen role
 inventory_playbooks:
   - duffy_inventory.yml
-
 async: false
 async_timeout : 1000
 no_output : false
 schema: "ex_schemas/schema_v2.json"
+inventory_layouts_path: "inventory_layouts"
+inventory_outputs_path: "inventory_outputs"
+inventory_type: "aws"
```

Comment:
I agree with you it doesnot work with multiple inventories , 
In that case we might have to loop it thorough a list of inventory types. 
Will update it in next commit .